File: jobs/2026-04-25__23-53-07/hatchet_wait_event__kmhc7sd/artifacts/code/worker.py
from workflow import workflow, hatchet

def main():
    # Create a worker
    worker = hatchet.worker("order-worker")
    
    # Register the OrderWorkflow
    worker.register_workflow(workflow)
    
    # Start the worker
    worker.start()

if __name__ == "__main__":
    main()

File: jobs/2026-05-26__11-16-44/concurrency_limit_ts__PMMAN5b/artifacts/user/myproject/src/worker.ts
import { hatchet, processOrder } from './hatchet.js';

const main = async () => {
  const worker = await hatchet.worker('process-order-worker', {
    workflows: [processOrder],
    slots: 10,
  });

  await worker.start();
};

main().catch((error) => {
  console.error('Worker failed to start:', error);
  process.exit(1);
});
